threadsd/main.go

```diff
@@ -28,7 +31,7 @@ var log = logging.Logger("threadsd")
 func main() {
 	fs := flag.NewFlagSetWithEnvPrefix(os.Args[0], "THRDS", 0)
 
-	repo := fs.String("repo", ".threads", "Repo location")
+	repo := fs.String("repo", ".threadsd", "Repo location")
```

Comment:
Isn't this a breaking change? An existing `threadsd` that uses the previous default value won't find its data and start from scratch. Was this intentional?

Reply:
Ah yeah, reverted!